crates/core/c8y_api/src/smartrest/operations.rs

```diff
@@ -119,6 +147,55 @@ impl Operation {
             .map(|exec| exec.forceful_timeout)
             .unwrap_or(DEFAULT_FORCEFUL_TIMEOUT)
     }
+
+    pub fn is_valid_opeation_handler(&self) -> bool {
```

Comment:
```suggestion
    pub fn is_valid_operation_handler(&self) -> bool {
```

Reply:
Oops. Done in 31913a47803f14de3bb1d2c6c5089cf223f3df14